shared/utils/combine-ovals.py

```diff
@@ -54,11 +48,12 @@ def _header(schema_version):
         http://oval.mitre.org/XMLSchema/oval-definitions-5#unix unix-definitions-schema.xsd
         http://oval.mitre.org/XMLSchema/oval-definitions-5#linux linux-definitions-schema.xsd">
     <generator>
-        <oval:product_name>python</oval:product_name>
-        <oval:product_version>%s</oval:product_version>
+        <oval:product_name>combine-ovals.py from SCAP Security Guide</oval:product_name>
+        <oval:product_version>ssg: %s, python: %s</oval:product_version>
```

Comment:
What is the point of having python in the product version? Shouldn't this only be the SSG product version?